deps/src/polymake_caller.cpp

```diff
@@ -89,8 +89,21 @@ pm::perl::PropertyValue polymake_call_method(std::string function_name, pm::perl
     return function();
 }
 
+void polymake_call_method_void(std::string function_name, pm::perl::Object object, jlcxx::ArrayRef<jl_value_t*> arguments)
+{
+    size_t argument_list = arguments.size();
+    auto function = object.prepare_call_method(function_name);
+    for(size_t i = 0;i<argument_list;i++){
+        polymake_call_function_feed_argument(function, arguments[i]);
+    }
+    function();
+    return;
```

Comment:
```suggestion
```
The return here is unnecessary

Reply:
Here you see that I have no idea about C++^^. I need a new approval now :)